framework/src/Volo.Abp.MongoDB/Volo/Abp/Domain/Repositories/MongoDB/MongoDbRepository.cs

```diff
@@ -484,6 +484,11 @@ public async override Task<long> GetCountAsync(CancellationToken cancellationTok
         await DeleteManyAsync(entities, autoSave, cancellationToken);
     }
 
+    public override async Task DeleteDirectAsync(Expression<Func<TEntity, bool>> predicate, CancellationToken cancellationToken = default)
+    {
+        await DeleteAsync(predicate, true, cancellationToken);
```

Comment:
Does't MongoDB have a way of doing that? If it has, please also properly implement it for MongoDB too.